src/components/translator/hts_pre_test_DRAFT.PT.json

```diff
@@ -0,0 +1,133 @@
+{
+    "name": "HTS Pre-Test Counselling",
```

Comment:
Not sure if i missed a reference to these json files but these should be removed if not needed along other unused json files